src/app/components/test/test.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { GetApiService } from 'src/app/get-api.service';
+
+@Component({
+  selector: 'app-test',
+  templateUrl: './test.component.html',
+  styleUrls: ['./test.component.css']
+})
+export class TestComponent implements OnInit {
+
+  public activity!: string;
+  constructor(private api: GetApiService){
+    api.apiCall().subscribe((data:any)=>{
+      this.activity = JSON.stringify(data, ['activity']);
```

Comment:
Can you try this?:
`this.activity = data.activity;`

Reply:
anotado

Reply:
it works 👍